include/circt/Dialect/SV/SVStatements.td

```diff
@@ -463,17 +463,20 @@ def VerbatimOp : SVOp<"verbatim"> {
 
     sv.verbatim allows operand substitutions with {{0}} syntax.
   }];
-  let arguments = (ins StrAttr:$string, Variadic<AnyType>:$operands);
+  let arguments = (ins StrAttr:$string, Variadic<AnyType>:$operands,
+                  DefaultValuedAttr<SymRefArrayAttr,"{}">:$symRefs);
 
   let assemblyFormat =
-    "$string attr-dict (`(` $operands^ `)` `:` type($operands))?";
+    "$string (`(` $operands^ `)` `:` type($operands))? attr-dict";
```

Comment:
Moving the attr-dict to the end makes sense no matter what, it would be great to get the symbols to have a nicer syntax tho. Does:

`"$string (`(` $operands^ `)` $symbols `:` type($operands))? attr-dict"`

work?  Does it look nice?

Reply:
Trying to figure out a way to make it look better, `"$string (($operands^)$symbols: type($operands))? attr-dict"` does not work!

Reply:
Ok, don't worry about over polishing this if it is a pain, we can always shine it up later if there is a reason thx